examples/all.yaml

```diff
@@ -533,4 +533,14 @@ openshift_test_private_e2e_repo: "https://github.com/openshift/openshift-tests-p
 openshift_test_private_git_branch: "master"
 openshift_test_private_directory: "/tmp/openshift_test_private_e2e"
 openshift_test_private_golang_tarball: "https://go.dev/dl/go1.24.1.linux-ppc64le.tar.gz"
-testcase_filters : 'Image_Registry\\|API_Server'
\ No newline at end of file
+testcase_filters : 'Image_Registry\\|API_Server'
+=======
+# ocp-odf-operator required vars
+odf_enabled: false
+update_channel: ""
+odf_catalogsource_image: "" #Example - quay.io/rhceph-dev/ocs-registry:4.16.3
+worker0_name: "worker-0"
+worker1_name: "worker-1"
+worker2_name: "worker-2"
```

Comment:
Can we get the first 3 workers automatically instead of variables?

Reply:
I see https://github.com/ocp-power-automation/ocp4-playbooks-extras/pull/116 repeating these variables.